src/earn/EarnEntrypoint.test.tsx

```diff
@@ -1,47 +1,146 @@
 import { fireEvent, render } from '@testing-library/react-native'
 import React from 'react'
+import { Provider } from 'react-redux'
 import AppAnalytics from 'src/analytics/AppAnalytics'
 import { EarnEvents } from 'src/analytics/Events'
 import EarnEntrypoint from 'src/earn/EarnEntrypoint'
+import { EarnTabType } from 'src/earn/types'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { getFeatureGate } from 'src/statsig'
 import { StatsigFeatureGates } from 'src/statsig/types'
+import { NetworkId } from 'src/transactions/types'
+import { createMockStore } from 'test/utils'
 
 jest.mock('src/statsig')
 
+function getStore({ hasSuppliedPools = false }: { hasSuppliedPools?: boolean } = {}) {
+  return createMockStore({
+    positions: {
+      positions: [
+        {
```

Comment:
nit: can we use mockEarnPositions from test/values?

Reply:
updated in [842c45f](https://github.com/valora-inc/wallet/pull/6233/commits/842c45f19c140afa5d33c7af357b9209f3ce997f)